syntax = "proto3";
package mars.stn;


option java_package = "win.liyufan.im.proto";
option java_outer_classname = "DismissGroupRequestOuterClass";

//option objc_class_prefix = "LYF";

import "message_content.proto";

message DismissGroupRequest {
  string group_id = 1;
  MessageContent notify_content = 2;
}
